api/ereqs_admin/static/admin/override.scss

```diff
@@ -46,3 +50,134 @@ $nav-menu-width: 200px;
   position: absolute;
   width: $nav-menu-width;
 }
+
+.app-admin_interface,
+.admin-interface {
+  background: #F0F0F0;
+}
+
+.object-tools a {
```

Comment:
When overriding styles within the admin, let's comment what they're changing. Otherwise, there's not a good way to figure out why these changes were made.